Marlin/Configuration.h

```diff
@@ -1498,6 +1498,40 @@
 //
 //#define OLED_PANEL_TINYBOY2
 
+//
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+//
+// https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html
+//
+// 2 line display.  
+//
+//#define MAKEBOARD_MINI_2_LINE_DISPLAY_1602
+
+//
+// MKS Mini Panel with graphic controller and SD support 
+//
+//http://reprap.org/wiki/MKS_MINI_12864
+//
+// Be aware that MKS/makebase/makeboard install the 8 pin EXP1 and EXP2 headers reversed on all their products.  
```

Comment:
Why mention this hear, since you are creating an entirely new panel entry for MINIPANEL that essentially flips the connectors logically ?  Doesn't this just add more confusion ?

Reply:
it does not flip power.